### DISCUSSION TITLE
kettle 连接 doris 进行数据对接很慢

### DISCUSSION THREAD

**Discussion by SkingWo** [apache/doris#35183] [A - General / Q&A]
kettle 连接doris 使用的是mysql驱动，但是读写很慢，平均每秒200,-300行记录读写，
网上查阅了一些资料，在数据库连接增加了配置项，rewriteBatchedStatements=true，开启doris批量提交，依然不行，有大神知道该如何解决么？

**Comment by yagagagaga** [comment]
中文问题，建议去 ask.selectdb.com 提问
